api/src/cli/commands/init/index.ts

```diff
@@ -49,19 +48,19 @@ export default async function init(): Promise<void> {
 			await runSeed(db);
 			await runMigrations(db, 'latest');
 		} catch (err: any) {
-			console.log();
-			console.log('Something went wrong while seeding the database:');
-			console.log();
-			console.log(`${chalk.red(`[${err.code || 'Error'}]`)} ${err.message}`);
-			console.log();
-			console.log('Please try again');
-			console.log();
+			logger.info('');
+			logger.info('Something went wrong while seeding the database:');
+			logger.info('');
+			logger.info(`${chalk.red(`[${err.code || 'Error'}]`)} ${err.message}`);
+			logger.info('');
+			logger.info('Please try again');
+			logger.info('');
```

Comment:
We can drop the `''` here. With `console.log`, it'll print an empty line, but with `pino`, it'll construct a whole new logging object without a message, which isn't what we'd want.

```suggestion
			logger.info(err, `
Something went wrong while seeding the database:

${chalk.red(`[${err.code || 'Error'}]`)} ${err.message}

Please try again
			`);
```